examples/Example14/include/HepMC3G4AsciiReader.hh

```diff
@@ -19,12 +19,16 @@ class HepMC3G4AsciiReaderMessenger;
 class HepMC3G4AsciiReader : public HepMC3G4Interface {
 protected:
   G4String filename;
+  int firstEventNumber = 0;
+  int maxNumberOfEvents = __INT_MAX__;
```

Comment:
Maybe safer to use `INT_MAX` after including `<climits>`? Not sure where `__INT_MAX__` is defined and if it is portable.